std_msgs/Header header

controller/Trajectory_point[] trajectory
string mode
int64 forceControl